transports/Dockerfile

```
@@ -1,64 +1,47 @@
 # Set environment for static build
-ENV CGO_ENABLED=0
-ENV GOOS=linux
-ENV GOARCH=amd64
+ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
 
 # Define build-time variable for transport type
 ARG TRANSPORT_TYPE=http
 
-# Initialize Go module and fetch the bifrost transport package
+# Initialize Go module, fetch, and build in fewer layers
 RUN go mod init bifrost-transports && \
```

Hadolint (2.12.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)